protocol/packages/swap/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license.workspace = true
 [package.metadata.cargo-each]
 combinations = [
     { tags = ["ci", "net_dev", "net_test", "net_main", "neutron-astroport-usdc_axelar"], feature-groups = ["dex", "astroport-net"], always-on = ["astroport"], include-rest = true },
+    { tags = ["ci", "net_main", "neutron-astroport-usdc_noble"], always-on = ["main", "astroport"], include-rest = false },
```

Comment:
It's missing the `testing` feature as `include-rest` is set to `false`.
Please add a feature group for it identical, if not the same, as with the other packages.

Reply:
nice catch!